bigchaindb/backend/rethinkdb/config.py

```diff
@@ -0,0 +1,48 @@
+"""Database configuration functions."""
+import logging
+from functools import partial
+
+import rethinkdb as r
+
+from bigchaindb.backend import config
+from bigchaindb.backend.exceptions import DatabaseOpFailedError
+from bigchaindb.backend.rethinkdb.connection import RethinkDBConnection
+
+logger = logging.getLogger(__name__)
+
+TABLES = ('bigchain', 'backlog', 'votes')
```

Comment:
Well, ok, then perhaps we put under the `rethinkdb` namespace for now, and I create an issue about this question.